olp-cpp-sdk-core/src/client/OlpClientSettingsFactory.cpp

```diff
@@ -42,7 +45,21 @@ OlpClientSettingsFactory::CreateDefaultNetworkRequestHandler(
 std::unique_ptr<cache::KeyValueCache>
 OlpClientSettingsFactory::CreateDefaultCache(cache::CacheSettings settings) {
   auto cache = std::make_unique<cache::DefaultCache>(std::move(settings));
-  cache->Open();
+  auto error = cache->Open();
+  if (error == cache::DefaultCache::StorageOpenResult::OpenDiskPathFailure) {
+    std::string disk_path_mutable = settings.disk_path_mutable
+                                        ? settings.disk_path_mutable.get()
+                                        : "(empty)";
+    std::string disk_path_protected = settings.disk_path_protected
+                                          ? settings.disk_path_protected.get()
+                                          : "(empty)";
+
+    OLP_SDK_LOG_FATAL_F(
+        kLogTag,
+        "Error opening disk cache. Mutable Path: %s Protected Path: %s",
```

Comment:
```suggestion
        "Error opening disk cache, disk_path_mutable=%s, disk_path_protected=%s",
```

To follow the same pattern as everywhere else.